taichi/program/async_utils.h

```diff
@@ -4,14 +4,13 @@
 #include <unordered_set>
 
 #include "taichi/ir/snode.h"
+#include "taichi/ir/statements.h"
```

Comment:
`taichi/ir/statements.h` includes `taichi/ir/ir.h`...

Reply:
True... Unfortunately i need this to pull in `OffloadedStmt::TaskType`